internal/utils/config.go

```diff
@@ -656,6 +659,10 @@ func LoadConfigFS(fsys afero.Fs) error {
 				}
 			}
 
+			if !IsValidTimeSecondLiteral(Config.Auth.Sms.MaxFrequency) {
```

Comment:
Let's assign a default value of `1s` on line 322 and 323. Otherwise this will break existing user's config.toml